### ISSUE TITLE
Only show specific hidden directories or files on input.

### ISSUE THREAD

**Issue by xTekC** [Canop/broot#551]
Add the ability to show only specific hidden directories and/or files on input, instead of showing every hidden directory and file.

Examples using (-ih flag):
`-ih, --input-hidden`

`br -ih "/hidden_dir"`

`br -ih "hidden_file" `

**Comment by Canop** [comment]
You mean you'd want to whitelist some specific paths for a broot session ?

**Comment by xTeKc** [comment]
Yeah, that would work in this instance.
Instead of showing all hidden, only show /.config
![brsc1](https://user-images.githubusercontent.com/81730792/168525276-b0a1e8d6-8c78-458e-9a4d-81e36d511cd9.png)

**Comment by Canop** [comment]
Wouldn't it be more convenient for you to just have the .config directory always shown ?

**Comment by xTeKc** [comment]
Yes, that would be more convenient. But, if I took a screenshot of the tree layout to use for reference, it would be nice to have the exact name and location of certain hidden directories and/or files without showing all the other unimportant hidden directories and/or files.

**Comment by Canop** [comment]
Showing chosen hidden files while keeping the others hidden is done with `special_paths` in conf.hjson, for example `"~/.config": { show: always }`. Launch flags to do that on the fly are discussed in #558. Closing.
